src/test/kotlin/no/nav/familie/ef/søknad/mock/DokumentController.kt

```diff
@@ -0,0 +1,45 @@
+package no.nav.familie.ef.søknad.mock
+
+import no.nav.security.token.support.core.api.Unprotected
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import org.springframework.web.multipart.MultipartFile
+import java.util.*
+
+@RestController
+@RequestMapping(path = ["/dokumentmock/"], produces = [MediaType.APPLICATION_JSON_VALUE])
+@Unprotected
+class DokumentController(private val dokumenter: MutableMap<String, ByteArray>) {
```

Comment:
Hvis vi har denne og endrer url i soknad frontend trenger vi ikke å starte opp familie-dokumenter også.. Min maskin sliter fort når jag har 4 applikasjoner igang allerede :P

Reply:
Det er greit for meg :)

Reply:
men denne tar ikke høyde for mellomlagring, kun vedlegg..?

Reply:
Legge til mockendepunkt for mellomlagring også?

Reply:
Done!